san/batch.py

```diff
@@ -16,7 +17,12 @@ def execute(self):
         batched_queries = []
 
         for idx, q in enumerate(self.queries):
-            batched_queries.append(get_gql_query(idx, q[0], **q[1]))
+            metric = q[0].split('/')[0]
+            slug = q[0].split('/')[1]
```

Comment:
```python
[metric, slug] = str.split("/")
```
to avoid splitting twice

Reply:
Also, what is this `q` exactly so we need to fetch the thing at position `0`?

Reply:
q[0] is the dataset, and q[1] are the kwargs, it can be seen in the get() method